dotrun.py

```
@@ -113,6 +130,33 @@ def _get_container_name(self, command=None):
         # Remove duplicated hyphens
         return re.sub(r"(-)+", r"\1", name)
 
+    def _set_additional_mounts(self, command=None):
+        """
+        Return a list of additional mounts
+        """
+        if not command:
```

Can `command` even be missing here? I would not bother to check. And also not default `command=None`. I assume something would need to go awfully wrong for it to get to this point and not have a `command`. I would let the thing error out.